src/overlays/actors/ovl_En_Ice_Hono/z_en_ice_hono.c

```diff
@@ -151,35 +151,37 @@ void EnIceHono_InitSmallFlame(Actor* thisx, PlayState* play) {
 
 void EnIceHono_Init(Actor* thisx, PlayState* play) {
     EnIceHono* this = (EnIceHono*)thisx;
-    s16 params = this->actor.params;
+    s16 params = this->actor.params; // Required for regalloc
```

Comment:
is there a point introducing these comments at this point?

Reply:
yeah imo this is not needed

Reply:
I hope everyone noticed that this temp is not actually used.

Reply:
I did not :sweat_smile: 
Now I would say either remove the comment anyway, or actually put an explanation since I assume we don't know for sure that's what the original source had?

Reply:
I would prefer `// unused but required to match`
reason being that we are moving toward this being less of a decomp codebase and more of a "useable" codebase, so the nitty gritty details like "regalloc" start to become less important